nexus/src/external_api/http_entrypoints.rs

```diff
@@ -3380,14 +3392,169 @@ async fn sshkeys_delete_key(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
-#[cfg(test)]
-mod test {
-    use super::external_api;
 
-    #[test]
-    fn test_nexus_tag_policy() {
-        // This will fail if any of the endpoints don't match the policy in
-        // ./tag-config.json
-        let _ = external_api();
-    }
+/// Path parameters for request by id 
+#[derive(Deserialize, JsonSchema)]
+struct ByIdPathParams {
+    id: Uuid,
+}
+
+
+/// Fetch an organization by id
+#[endpoint {
+    method = GET,
+    path = "/by_id/organizations/{id}",
+    tags = ["organizations"],
+}]
+async fn by_id_organization_get(
```

Comment:
A few things notes
- I preserved the tags for the resources it maps to. I'm assuming that's what we'd want.
- I left the resource plural in the route to match with how it exists in other parts of the API. It's somewhat weird in that it's only used to give you a single item, but it seemed prudent given this could be used for batching in the future
- I grouped all these together because they share the same parent route despite them returning different resources